dockyard/common/information/information.py

```diff
@@ -0,0 +1,21 @@
+from dockyard.common import base, link
+from dockyard.common import utils
+
+class Information(object):
+    def __init__(self):
+        self.rest_client = base.RESTClient()
+
+    def info(self):
+        host = utils.get_host()
+        ln = link.make_url(host=host, protocol='http', url='/info')
+        return self.rest_client.GET(ln).data
+
```

Comment:
I think, as we will be using our application therefor we need to add information and version of our application in addition to the information of the docker daemon.